src/routes/news/containers/LocalNewsPage.js

```diff
@@ -48,7 +49,10 @@ export class LocalNewsPage extends React.Component<PropsType> {
       return <LoadingSpinner />
     }
 
-    const currentCity: CityModel = cities && cities.find(cityElement => cityElement.code === city)
+    const currentCity: ?CityModel = cities && cities.find(cityElement => cityElement.code === city)
+    if (!currentCity) {
```

Comment:
This should be handled already, it's because of flow, right?

Reply:
correct. `find` on an array can always return null. Also you wouldn't know that it's already handled only looking at this file I guess.